onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_support_checker.cc

```diff
@@ -2183,6 +2183,98 @@
 
 #pragma endregion
 
+#pragma region op_pad
+
+class PadOpSupportChecker : public BaseOpSupportChecker {
+ private:
+  int32_t GetMinSupportedNNAPIFeatureLevel(const NodeUnit& /* node_unit */,
+                                           const OpSupportCheckParams& /* params */) const override {
+    return ANEURALNETWORKS_FEATURE_LEVEL_3;  // for ANEURALNETWORKS_PAD_V2
+  }
+
+  int GetMinSupportedOpSet(const NodeUnit& /* node_unit */) const override {
+    return 11;  // before Pad-11, inputs `pads` and `constant_value` were attributes
+  }
```

Comment:
Attributes are essentially constant initializers so this shouldn't prevent supporting the older opset should it?

Reply:
right, it could be supported. actually support could be expanded in several ways, but this initial change has the necessary support for the mobilebert models.

Reply:
would be great for the comment to say that's why. right now if I read the comment I would assume it's not possible.

Reply:
added some comments about what we can add support for later